sign in with username and password (#47)
tidbcloud/dbaas-docs: manage-user-access.md

## Sign in

> Navigate to the TiDB Cloud login page: <https://tidbcloud.com>.

Reviewer: ```suggestion
1. Navigate to the TiDB Cloud login page: <https://tidbcloud.com>.
```

> Enter your user name and password, and click sign in.

Reviewer: Please use the exact words and capitalization as those in UI.
```suggestion
2. Enter your user name and password, and click **Sign In**.
```

> Enter your user name and password, and click sign in.

Reviewer: Another question: I see that on the UI, it displays as "Email" and "Password". We'd better keep wording consistent with UI, to avoid making user confused.

Author: yes, please change it to Email and Password.

> If you already created a TiDB Cloud account using your Google account, sign in to your Google account.

Reviewer: ```suggestion
If you already created a TiDB Cloud account using your Google account, click **Sign in with Google**.
```
